plugins/out_cloudwatch_logs/cloudwatch_logs.c

```diff
@@ -689,6 +723,12 @@ static struct flb_config_map config_map[] = {
      "Specify the log storage class. Valid values are STANDARD (default) and INFREQUENT_ACCESS."
     },
 
+    {
+    FLB_CONFIG_MAP_BOOL, "add_entity", "false",
+    0, FLB_TRUE, offsetof(struct flb_cloudwatch, add_entity),
+    "add entity to PutLogEvent calls"
```

Comment:
We will want to add documentation for the new parameter to the new output plugins - https://github.com/fluent/fluent-bit-docs/blob/master/pipeline/outputs/cloudwatch.md#configuration-parameters